packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/BackgroundStyleApplicator.kt

```diff
@@ -49,6 +50,13 @@ public object BackgroundStyleApplicator {
     ensureCSSBackground(view).color = color ?: Color.TRANSPARENT
   }
 
+  @JvmStatic
```

Comment:
1. We might eventually support background images based on URI, in which case we’d need to change this public API. Can we design in a way that will be future proof to that? Like maybe having a type for background image layer, that can be gradient type now, and uri later?
2. Can we accept by list instead of array for consistency?